docs/windows.md

```diff
@@ -46,3 +46,18 @@ the explorer window by hitting `<cmd>+R` and type in:
 - `explorer %LOCALAPPDATA%\Programs\Ollama` contains the binaries (The installer adds this to your user PATH)
 - `explorer %HOMEPATH%\.ollama` contains models and configuration
 - `explorer %TEMP%` contains temporary executable files in one or more `ollama*` directories
+
+
+## Multi-user Advanced Scenarios
+
+The `OllamaSetup.exe` installer has been designed for individual use on your
+Windows PC.  It installs in your account without requiring Administrator rights.
+We update Ollama regularly to support the latest models, and this installer will
+help you keep up to date.
+
+If you have a multi-user system, or are looking for more control over where
+Ollama is installed, and want to run it as a system service, we also provide an
+`ollama-windows-amd64.zip` file which contains the ollama CLI and GPU library
+dependencies for Nvidia and AMD.  This does not include the tray application, or
+update logic to keep your install current.  You can use a tool like
```

Comment:
Maybe:

Use tools such as [NSSM](https://nssm.cc/) to run `ollama serve` as a system service.